tests/bde/test_bde.py

```diff
@@ -271,3 +271,24 @@ def test_bde_eow_partial(bde_eow_partial: BinaryIO) -> None:
 
 def test_is_bde_volume(bde_aes_128: BinaryIO) -> None:
     assert is_bde_volume(bde_aes_128)
+
+
+def test_bde_invalid_keys() -> None:
```

Comment:
Maybe "wrong" is better than "invalid" here, because BitLocker keys can also be invalid in general, because they do not adhere to the exact pattern